source/_components/sensor.monzo.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: "Monzo Bank"
+description: "How to integrate your Monzo Bank account within Home Assistant."
+date: 2018-09-07 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: "monzobank.png"
+ha_category: Finance
+featured: false
+ha_release: "0.79"
```

Comment:
This got to be `"0.80"`.

Reply:
Fixed in new commit. Thanks.

Reply:
Not correct, @Solije @WofWca.
The "0.80" already has been cut off (feature stop/beta).
So the first possible one for this to get into would be `0.81`